PickMeUp/PickMeUp/PMLoginViewController.m

```diff
@@ -0,0 +1,62 @@
+//
+//  LoginViewController.m
+//  PickMeUp
+//
+//  Created by Abigail Shilts on 7/5/22.
+//
+
+#import "PMLoginViewController.h"
+#import "Parse/Parse.h"
+#import "StringsList.h"
+
+
+@interface PMLoginViewController ()
+@property (weak, nonatomic) IBOutlet UITextField *insertedUsername;
+@property (weak, nonatomic) IBOutlet UITextField *insertedPassword;
+
+@end
+
+@implementation PMLoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+
+}
+
+
+
+- (void)loginUser {
+    // Pop up alert to ensure user enters all fields
+    UIAlertController *alert = [UIAlertController alertControllerWithTitle:missingFields message:logginginNeedsAll preferredStyle:(UIAlertControllerStyleAlert)];
```

Comment:
You should create alerts controller only if necessary. Should be moved into if {} below